Generators/src/main/java/io/deephaven/pythonPreambles/plotV2.py

```diff
@@ -3,7 +3,7 @@
 #
 
 ######################################################################################################################
-#               This code is auto generated. DO NOT EDIT FILE!
+#               Preample code that gets added to generated code in GeneratePyV2FigureAPI.java
```

Comment:
```suggestion
#               Preamble code that gets added to generated code in GeneratePyV2FigureAPI.java
```

Reply:
Changing this header causes problems elsewhere.  With this change, the header in `figure.py` is now wrong.  It is clearly confusing, but it was probably better before this change.  To fix it properly would require a change to the generator.  Not sure if it is worth that right now.